spec/acceptance/machine_delete_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'spec_helper_acceptance'
 require 'securerandom'
+require 'spec_helper_acceptance_local'
```

Comment:
this should not be needed, as `spec_helper_acceptance` is already requiring this